parse_csv.py

```diff
@@ -0,0 +1,74 @@
+import csv
+import sys
+
+def strip_prefix(input_string, prefix):
+    if input_string.startswith(prefix):
+        return input_string[len(prefix):]
+    return input_string
+
+def add_to_output(export_output, row, prefix):
+    if row['URL'].startswith(prefix):
+        endurl = strip_prefix(row['URL'], prefix)
+        if endurl in export_output:
+            old_row = export_output[endurl]
+            for col in row.keys():
+                if col == "URL":
+                    row[col] = "https://www.parse.ly" + endurl
+                    continue
+                try:
+                    if old_row[col] != "":
```

Comment:
tbh i'm not sure if the floats really give you much here. since you are not doing any division, ints should be just fine